### PR TITLE
Fetch and display workouts

### LINKED ISSUE #15: Fetch, parse, and display recent workouts
Display info about my most recent three workouts

### PR DESCRIPTION
**Pull request by daltonclaybrook**
Resolves #15

### CODE REVIEW

File: src/components/Workouts/WorkoutCard.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React, { FunctionComponent } from 'react';
+import Workout from '../../models/Workout';
+
+const WorkoutCard: FunctionComponent<Workout> = (workout) => (
+    <div className="workout">
+        <h3>{workout.name}</h3>
```
Comment (drewwyatt) [inline src/components/Workouts/WorkoutCard.tsx]:
```suggestion
        <h3>{name}</h3>
```

File: src/components/Workouts/Workouts.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { fetchWorkouts } from './api';
+import WorkoutCard from './WorkoutCard';
+
+interface WorkoutsExpecting {
+    workouts: Workout[];
+}
+
+const enhancer = (Component: FunctionComponent<WorkoutsExpecting>) => () => {
+    const [workouts, loadWorkouts] = useState<Workout[]>([]);
+    useEffect(() => {
```
Comment (drewwyatt) [inline src/components/Workouts/Workouts.tsx]: I see some duplication between books, checkins, and workouts. Do you think it's possible to DRY up some of the boilerplate?

**Comment by drewwyatt** [comment]
Looks great, all comments here are minor and can either be iterated on later, or ignored entirely.

File: src/components/Workouts/WorkoutCard.tsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React, { FunctionComponent } from 'react';
+import Workout from '../../models/Workout';
+
+const WorkoutCard: FunctionComponent<Workout> = (workout) => (
```
[the remark this answers is not in the document]
Comment (daltonclaybrook) [reply src/components/Workouts/WorkoutCard.tsx]: What's your threshold for count of props before you just use the whole model?

File: src/components/Workouts/Workouts.tsx (hunk 2 above)
Comment (daltonclaybrook) [reply src/components/Workouts/Workouts.tsx]: Completely agree about this. I will try this in a new PR.

**Pull request merged**